mandatory/1-fix-functions.js

```diff
@@ -21,18 +19,18 @@ function mood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = num;
 
-  if (isBigEnough) {
+  if (isBigEnough>10) {
     return "num is greater than 10";
   } else {
     return "num is not big enough";
   }
 }
 
 function get3rdIndex(arr) {
-  let index = 3;
-  let element;
+  let index = arr[3];
+  let element = index;
```

Comment:
here I think it is nicer if we do `let element = arr[index]` and don't change index.

Your approach works too